exercises/word-search/word_search_test.py

```diff
@@ -2,9 +2,112 @@
 
 from word_search import WordSearch, Point
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
+
```

Comment:
Minor formatting point, but can you move the version comment down a line (ie. two blank lines before, one after).